<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimatePackerClass</class>
 <widget class="QMainWindow" name="AnimatePackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>AnimatePacker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Plists</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBar" name="toolBar_5">
           <property name="windowTitle">
            <string>toolBar_5</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <addaction name="actionOpenPlist"/>
           <addaction name="actionDeletePlist"/>
          </widget>
         </item>
         <item>
          <widget class="PlistList" name="plistList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Animations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBar" name="toolBar_2">
           <property name="windowTitle">
            <string>toolBar_2</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="floatable">
            <bool>true</bool>
           </property>
           <addaction name="actionAnimation"/>
           <addaction name="actionCopy"/>
           <addaction name="actionDeleteAnimation"/>
          </widget>
         </item>
         <item>
          <widget class="AnimationsTable" name="animationTable">
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Delay</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>FlipX</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>FlipY</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBar" name="toolBar_3">
           <property name="windowTitle">
            <string>toolBar_3</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <addaction name="actionPlay"/>
           <addaction name="actionStop"/>
           <addaction name="actionZoomIn"/>
           <addaction name="actionZoomOut"/>
           <addaction name="actionPalette"/>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>301</width>
              <height>114</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="imageLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>SpriteFrames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBar" name="toolBar_4">
           <property name="windowTitle">
            <string>toolBar_4</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <addaction name="actionMoveUp"/>
           <addaction name="actionMoveDown"/>
           <addaction name="actionDeleteSpriteFrame"/>
          </widget>
         </item>
         <item>
          <widget class="SpriteFramesList" name="spriteFramesList">
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget4">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>Sprites</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolBar" name="toolBar_6">
          <property name="windowTitle">
           <string>toolBar_6</string>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <addaction name="actionChoose"/>
          <addaction name="actionListMode"/>
          <addaction name="actionIconMode"/>
         </widget>
        </item>
        <item>
         <widget class="SpritesList" name="spritesList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlists">
    <property name="title">
     <string>Plists</string>
    </property>
    <addaction name="actionOpenPlist"/>
    <addaction name="actionDeletePlist"/>
   </widget>
   <widget class="QMenu" name="menuAnimations">
    <property name="title">
     <string>Animations</string>
    </property>
    <addaction name="actionAnimation"/>
    <addaction name="actionCopy"/>
    <addaction name="actionDeleteAnimation"/>
   </widget>
   <widget class="QMenu" name="menuPreview">
    <property name="title">
     <string>Preview</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionPalette"/>
   </widget>
   <widget class="QMenu" name="menuSpriteFrames">
    <property name="title">
     <string>SpriteFrames</string>
    </property>
    <addaction name="actionMoveUp"/>
    <addaction name="actionMoveDown"/>
    <addaction name="actionDeleteSpriteFrame"/>
   </widget>
   <widget class="QMenu" name="menuSprites">
    <property name="title">
     <string>Sprites</string>
    </property>
    <addaction name="actionChoose"/>
    <addaction name="actionListMode"/>
    <addaction name="actionIconMode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlists"/>
   <addaction name="menuAnimations"/>
   <addaction name="menuPreview"/>
   <addaction name="menuSpriteFrames"/>
   <addaction name="menuSprites"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/newObject.png</normaloff>:/image/newObject.png</iconset>
   </property>
   <property name="text">
    <string>New Xml</string>
   </property>
   <property name="toolTip">
    <string>New Xml</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Xml</string>
   </property>
   <property name="toolTip">
    <string>Save Xml</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Xml</string>
   </property>
   <property name="toolTip">
    <string>Open Xml</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/animation.png</normaloff>:/image/animation.png</iconset>
   </property>
   <property name="text">
    <string>Create Animation</string>
   </property>
   <property name="toolTip">
    <string>Create Animation</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/play.png</normaloff>
     <normalon>image/play.png</normalon>:/image/play.png</iconset>
   </property>
   <property name="text">
    <string>Play Animation</string>
   </property>
   <property name="toolTip">
    <string>Play Animation</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/stop.png</normaloff>:/image/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Animation</string>
   </property>
   <property name="toolTip">
    <string>Stop Animation</string>
   </property>
  </action>
  <action name="actionDeletePlist">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/deleteObject.png</normaloff>:/image/deleteObject.png</iconset>
   </property>
   <property name="text">
    <string>Delete Plist</string>
   </property>
   <property name="toolTip">
    <string>Delete Plist</string>
   </property>
  </action>
  <action name="actionDeleteAnimation">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/deleteObject.png</normaloff>:/image/deleteObject.png</iconset>
   </property>
   <property name="text">
    <string>Delete Animation</string>
   </property>
   <property name="toolTip">
    <string>Delete Animation</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/up.png</normaloff>:/image/up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/down.png</normaloff>:/image/down.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionDeleteSpriteFrame">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/deleteObject.png</normaloff>:/image/deleteObject.png</iconset>
   </property>
   <property name="text">
    <string>Delete SpriteFrame</string>
   </property>
   <property name="toolTip">
    <string>Delete SpriteFrame</string>
   </property>
  </action>
  <action name="actionChoose">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/choose.png</normaloff>:/image/choose.png</iconset>
   </property>
   <property name="text">
    <string>Choose Sprite</string>
   </property>
   <property name="toolTip">
    <string>Choose Sprite</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/zoomIn.png</normaloff>:/image/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/zoomOut.png</normaloff>:/image/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOpenPlist">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Plist</string>
   </property>
   <property name="toolTip">
    <string>Open Plist</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/palette.png</normaloff>:/image/palette.png</iconset>
   </property>
   <property name="text">
    <string>Palette</string>
   </property>
   <property name="toolTip">
    <string>Palette</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/copy.png</normaloff>:/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy Animation</string>
   </property>
   <property name="toolTip">
    <string>Copy Animation</string>
   </property>
  </action>
  <action name="actionIconMode">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/iconMode.png</normaloff>:/image/iconMode.png</iconset>
   </property>
   <property name="text">
    <string>Icon Mode</string>
   </property>
   <property name="toolTip">
    <string>Icon Mode</string>
   </property>
  </action>
  <action name="actionListMode">
   <property name="icon">
    <iconset resource="AnimatePacker.qrc">
     <normaloff>:/image/listMode.png</normaloff>:/image/listMode.png</iconset>
   </property>
   <property name="text">
    <string>List Mode</string>
   </property>
   <property name="toolTip">
    <string>List Mode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationsTable</class>
   <extends>QTableWidget</extends>
   <header>animations_table.h</header>
  </customwidget>
  <customwidget>
   <class>SpritesList</class>
   <extends>QListWidget</extends>
   <header>sprites_list.h</header>
  </customwidget>
  <customwidget>
   <class>PlistList</class>
   <extends>QListWidget</extends>
   <header>plist_list.h</header>
  </customwidget>
  <customwidget>
   <class>SpriteFramesList</class>
   <extends>QListWidget</extends>
   <header>sprite_frames_list.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AnimatePacker.qrc"/>
 </resources>
 <connections/>
</ui>
